.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+
+os:
+  - osx
+  - linux
+
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
+install:
+  - npm install
+  - npm run vscode:prepublish
```

Comment:
I think this prepublish and tsc are doing the same thing (I've got AppVeyor working only with the prepublish) so I'm gonna try tweaking that. Think this is all working now across both Travis and AppVeyor. I'll merge some tweaks to master shortly.

Thanks!